swarm/network/kademlia.go

```diff
@@ -492,6 +495,10 @@ func (k *Kademlia) neighbourhoodDepth() (depth int) {
 	return depth
 }
 
+func (k *Kademlia) neighbourhoodDepth() (depth int) {
```

Comment:
Why do we need this now? Can't we just call `depthForPot` directly by `kademlia.NeighbourhoodDepth`?